### DISCUSSION TITLE
Networked IOT weather radios

### DISCUSSION THREAD

**Discussion by bonelifer** [jamieden/dsame3#3] [Ideas]
You mentioned it in the issue from the original repo.  I'd say, ESP32 with an I2S DAC. You could use MPD(Music player daemon) on the main device(control server) to create a playlist for each type of warning. Then each networked device could be subscribed to only the relevant ones for that devices user.  There are MPD clients for ESP32 on GitHub.  A simply endpoint proxy(right terminology?) on each ESP32 could take a endpoint request, that request would tell it which playlist to broadcast, plus how many times it should repeat the message for that device.  Possibly using MQTT since that's something that should have a library for ESP32. That would also allow auto-magically finding the devices on the network. The main device(control server) would have a webui to allow the scheduling of which warnings to send to each device and how many times that user wants it repeated. Possibly even setting the volume for each devices, possibly depending on time of day.

**Comment by bonelifer** [comment]
The playlist would be like:
severWeatherTypeDSAMEcode

Then an network device could subscribe to multiple DSAME areas for specific weather types. For example, you'd only care about severe thunder storms in your DSAME area, but might care about a TORNADO warning/watch from an area that is adjacent to yours.

**Reply by jamieden** [reply]
I thought about that, but in a more broad sense. For example, all networked devices could stream to a central server so users could also receive alerts for other states. This is a great use case if you have relatives out in a different state, or if you have elderly family who aren't tech savvy. You could be their early warning system. I like to call it the SeWerNET, or the Severe Weather Network (the name could use some work).

**Reply by jamieden** [reply]
Although in terms of integrating it with an IoT device, further testing is required. I'd want to integrate it to be a full on weather radio, with enhanced smart features, like time synchronization using GPS, and a simplified GUI with advanced features that everyone can use.
